### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: templates/taxi/car_delete.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete car</h1>
+  <p>Are you sure, you want to delete car: {{ car }}</p>
+  <form action="" method="post">
+  {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="bth bth-danger">
+  </form>
+{% endblock content %}
```
Comment (pavlejviki) [inline templates/taxi/car_delete.html:9]: add a blank line

File: templates/taxi/car_form.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <form action="" method="post">
+    <h1>{{ object|yesno:"Update,Create" }} car</h1>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input class="bth btn-primary" type="submit" value="Submit">
+  </form>
+{% endblock content %}
```
Comment (pavlejviki) [inline templates/taxi/car_form.html:10]: add a blank line

File: templates/taxi/manufacturer_delete.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete manufacturer</h1>
+  <p>Are you sure, you want to delete manufacturer: {{ manufacturer }}</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="bth bth-danger">
+  </form>
+{% endblock content %}
```
Comment (pavlejviki) [inline templates/taxi/manufacturer_delete.html:9]: add a blank line

File: templates/taxi/manufacturer_form.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <form action="" method="post">
+  {% csrf_token %}
+  {{ form|crispy }}
+    <input type="submit" value="Submit  " class="bth bth-danger">
+  </form>
+{% endblock content %}
```
Comment (pavlejviki) [inline templates/taxi/manufacturer_form.html:9]: add a blank line

**Review by pavlejviki — CHANGES_REQUESTED** [review]
Attach screenshots of all created or modified pages to pull request. It's important to attach images not links to them.

**Review by vsmutok — APPROVED** [review]
Good Job 👍

**Pull request closed without merging**